pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -721,7 +735,22 @@ func (r *ReconcileLogStorage) shouldApplyElasticTrialSecret(ctx context.Context)
 	return false, nil
 }
 
-func (r *ReconcileLogStorage) kibanaSecrets(ctx context.Context) ([]*corev1.Secret, error) {
+// createCaCert is a convenience method for creating a secret that contains the ca to trust.
+func createCaCertSecret(caPem []byte, secretName string, namespace string) *corev1.Secret {
```

Comment:
I suggest moving this into a utils package.

Reply:
I have already done this in the next PR that is about to follow. Please ignore for now.